filter_lists/list_catalog.json

```diff
@@ -206,7 +206,7 @@
     },
     {
         "uuid": "67E792D4-AE03-4D1A-9EDE-80E01C81F9B8",
-        "title": "Fanboy's Annoyances",
+        "title": "Fanboy's Annoyances (and combined with uBO Annoyances list)",
```

Comment:
Maybe `Fanboy's Annoyances + uBO Annoyances`... or just `Annoyances`?

@rmcfadden3 any opinions here?

Reply:
`Fanboy's Annoyances + uBO Annoyances` works

Reply:
Yeah, agreed: `Fanboy's Annoyances + uBO Annoyances` works just fine.

The only question I might have is whether people know what `uBO` stands for. If there's any concern around that, you could expand to `Fanboy's Annoyances + uBlock Annoyances`. But honestly either option should work.

Reply:
Good point - `Fanboy's Annoyances + uBlock Annoyances` sounds good. The original entry was `uBlock Annoyances` anyways.

Reply:
Adjusted